crates/torii/cli/src/args.rs

```diff
@@ -148,9 +163,6 @@ impl TryFrom<ToriiArgs> for ToriiArgsConfig {
     type Error = anyhow::Error;
 
     fn try_from(args: ToriiArgs) -> Result<Self> {
-        // Ensure the config file is merged with the CLI arguments.
-        let args = args.with_config_file()?;
-
```

Comment:
Is this done elsewhere? Or we don't want to merge anymore?

Reply:
This is not used elsewhere. Had to remove it as it leads to recursion hell

Reply:
https://github.com/cartridge-gg/slot/blob/e694227a119bf38b4aed22bc16b83580c58e6cd2/cli/src/command/deployments/create.rs#L71

Used by slot for instance, but will check if we can't directly use the `with_config_file` as you mentioned to avoid recursion.